src/unix/qnx.c

```diff
@@ -67,18 +77,45 @@ int uv_exepath(char* buffer, size_t* size) {
 }
 
 
+static uint64_t uv__read_pidin_info(const char* what) {
+  uint64_t rc;
+  char* p;
+  char buf[BUF_SIZE];
+  const char* cmd = "pidin info";
+
+  FILE* fp = popen(cmd, "r");
+  if(fp == NULL)
```

Comment:
Style, here and elsewhere:
```suggestion
  if (fp == NULL)
```